scripts/cpplinter.sh

```diff
@@ -13,18 +13,15 @@ FILES_TO_PARSE=`find $SCRIPTDIR/.. -iname *.cpp -o -iname *.h -o -iname *.c`
 RET=0
 
 EXCLUDED_PATTERNS="\
-    ecc_enclave/_build \
     tlcc_enclave/nanopb \
-    tlcc_enclave/_build \
-    examples/auction/_build \
-    examples/echo/_build \
     common/protobuf \
     common/base64 \
     common/json \
     enclave_u.c \
     enclave_u.h \
     enclave_t.c \
     enclave_t.h \
+    _build \
```

Comment:
wouldn't it be safe to use /_build as pattern?  (we might well eventually add files foo_bar_build.c or alike eventually?

Reply:
yes! this works! I move this change to a separate PR!